Dockerfile.template

```diff
@@ -8,6 +8,15 @@ FROM resin/resinos:${OS_VERSION}.dev-${DEVICE_TYPE}
 COPY ./entry.sh /entry.sh
 COPY ./conf/systemd-watchdog.conf /etc/systemd/system.conf.d/watchdog.conf
 
+
+RUN sed -i 's/hosts:.*/hosts:\t\tfiles myhostname mdns4 mdns dns \[NOTFOUND=return\]/g' /etc/nsswitch.conf
+RUN echo -e '.local.\n.local\n'> /etc/mdns.allow
+
+COPY ./bob.local.pem /usr/share/ca-certificates/
```

Comment:
can we just mount in the certificate? or are updating the conf file and running `update-ca-certificates` necessary for it to be picked up?

Reply:
When ot running `update-ca-certificates` the self-signed certificate is not trusted to fetch the os-config.

Reply:
moved into a bob specific dockerfile